gcp_variant_transforms/testing/integration/run_tests.py

```diff
@@ -175,6 +228,14 @@ def _get_args():
   parser.add_argument('--staging_location', required=True)
   parser.add_argument('--temp_location', required=True)
   parser.add_argument('--logging_location', required=True)
+  parser.add_argument('--keep_tables',
+                      help='If set, created tables are not deleted.',
+                      action='store_true')
+  parser.add_argument('--revalidate_dataset',
```

Comment:
Consider renaming to revalidate_dataset_id (with an example in the comment) to clarify that it's the path and not a flag.